tests/unit/package_managers/yarn_classic/test_workspaces.py

```diff
@@ -71,7 +76,6 @@ def test_extracting_workspace_globs_works_for_all_types_of_workspaces(
     package: dict,
     expected: list,
 ) -> None:
-
```

Comment:
This hunk feels off, it's not connected to any other contextual change that would justify it (like the others in this patch), so let's leave the whitespace in there for now :), #_clean_diffs.

Reply:
Those lines are removed by vscode format on save feature

Reply:
Is vscode the suggested editor/IDE by this project? Can it be configured? I know this is just a single line, so not a big deal, but next time it could be a bigger change. I have a similar feature in vim, but configured only for trailing whitespaces which once got me in similar situation when I contributed a simple fix to some project with no formatting checks nor guidelines resulting in several hundred changes across the code base. So, one needs to look at the bigger scale of the consequences of these features, I don't think that "It's a feature of my editor" is an acceptable justification of irrelevant changes in one's diff regardless of quantity.

It's also a reason why I'm wary of blindly replacing `black` with `ruff` because without much configuration it reformatted the whole code base which is completely unjustifiable from git history POV, because it implies extra work for git blame to get to the actual commit adding/removing content.